FirebaseSessions/Tests/TestApp/Podfile.lock

```diff
@@ -1,118 +0,0 @@
-PODS:
```

Comment:
I think we'll still need this file? Maybe just revert it instead of deleting it

Reply:
This file got added by mistake in one of the previous PRs. With Podfile.lock, as a part of test app generation - we would be often forced to do a Pod repo update (if there are any version changes in the repo) which is annoying. So, reverted the addition of the file.